3p/integration.js

```diff
@@ -337,7 +335,10 @@ function triggerResizeRequest(width, height) {
 }
 
 function triggerRenderStart() {
-  nonSensitiveDataPostMessage('render-start');
+  if (!window.context.isRenderStartSent) {
```

Comment:
I disagree, this isn't private state for us to keep, this is a public object for the ad code to call and use. Providing this publicly seems unnecessary since it won't break our code internally (`render-start` is only listened for once, then it'll unregister the listener).